templates/foreman_gutterball.yaml.erb

```diff
@@ -0,0 +1,2 @@
+<%# for foreman_gutterball plugin %>
+:url: <%= @fqdn %>
```

Comment:
this should have `/gutterball` on the end. other than that, i will see if i can test this on monday!